packages/web-api/src/response/ViewsOpenResponse.ts

```diff
@@ -86,6 +86,7 @@ export interface Element {
   min_length?:                      number;
   max_length?:                      number;
   dispatch_action_config?:          DispatchActionConfig;
+  focus_on_load?:                   boolean;
```

Comment:
Good news, was able to find this on the docs site here: https://api.slack.com/reference/block-kit/block-elements
Bad news, not on the API methods example response for `views.open`: https://api.slack.com/methods/views.open - but I guess it would be too verbose to fully document all the possible Block elements on this page? What do you think?

Reply:
Yeah, I don't think the `views.open` API method page should cover all the patterns of blocks. If we would like to do so, it should mention not only focus_on_load but also many things such as response_url_enabled, and dispatch_action_config. As the page has a link to the modal document, I don't think we should repeat all the things there.